presto-hive/src/main/java/io/prestosql/plugin/hive/HdfsConfigurationUpdater.java

```diff
@@ -154,10 +153,9 @@ public static void configureCompression(Configuration config, HiveCompressionCod
         config.setBoolean("mapred.output.compress", compression);
         config.setBoolean(FileOutputFormat.COMPRESS, compression);
         // For DWRF
-        config.set(HIVE_ORC_DEFAULT_COMPRESS.varname, compressionCodec.getOrcCompressionKind().name());
-        config.set(HIVE_ORC_COMPRESSION.varname, compressionCodec.getOrcCompressionKind().name());
+        com.facebook.hive.orc.OrcConf.setVar(config, HIVE_ORC_COMPRESSION, compressionCodec.getOrcCompressionKind().name());
```

Comment:
What is `com.facebook.hive.orc.OrcConf`? Why `config` no longer work?

Reply:
It is DWRF, Facebook's fork for ORC. The old `HIVE_ORC_DEFAULT_COMPRESS` doesn't exist in Hive 2.x